### ISSUE TITLE
RM3 stop words

### ISSUE THREAD

**Issue by Peilin-Yang** [lintool/Anserini#116]
The RM3 reranking is called at https://github.com/lintool/Anserini/blob/master/src/main/java/io/anserini/search/SearchWebCollection.java#L202 where the constructor takes a stop words list file as the argument. 
I am not sure how the file was generated - but my guess is it was extracted from the index using https://github.com/lintool/Anserini/blob/master/src/main/java/io/anserini/util/ExtractRm3Stopwords.java (although there are 500 terms in https://github.com/lintool/Anserini/blob/master/src/main/resources/io/anserini/rerank/rm3/rm3-stoplist.gov2.txt whereas the default output size is 100)
If this is true then the stop words list is collection dependent. 
@lintool do you think it would affect the results of other collections? i.e. CW09B, CW12-b13, CW12-full.
This is the reason that why I did not test the RM3 on earlier TREC collections and any clarification will be appreciated.

**Comment by lintool** [comment]
That is correct - I have not done any stopword tuning (e.g., number of terms, etc.), and I've just used the gov2 terms for all the other RM3 runs. Please feel free to tune...

**Comment by lintool** [comment]
With the recent implementation of regression tests, it is perhaps time to revisit this issue.
